tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -1121,19 +1122,48 @@ public function theUserTriesToCreateASubFolderUsingTheGraphApi(
 	 *
 	 * @throws GuzzleException
 	 */
-	public function theUserHasCreateAFolderUsingTheGraphApi(
+	public function theUserHasCreatedAFolderUsingTheGraphApi(
```

Comment:
let's try to make similar to step name
```suggestion
	public function theUserHasCreatedAFolderInSpace(
```